### DISCUSSION TITLE
Best practice for dependent changes on merge requests

### DISCUSSION THREAD

**Discussion by andorfermichael** [renovatebot/renovate#10730] [Request Help] [answered]
What is the best approach for MRs which rely on further changes?

For example, a major version upgrade that needs adaptions of config etc. to keep the build, tests etc. working?

Can/Should I push the changes to the renovate MR or could this cause unexpected implications on renovate?

**Comment by viceice** [comment, marked as answer]
You can push any required changes to the renovate branch, but make sure to use a different git author, so renovate can detect the changes.
Renovate will only check latest branch commit, so make sure to not have renovate bot as last author on modified renovate branches.

So also don't ammend and force-push renovate commits, as the changes wouldn't detected by renovate

**Reply by andorfermichael** [reply]
Great. Thank you. Will do that.

**Reply by rarkins** [reply]
FYI pushing changes by a different git author will mean Renovate stops automatically updating the branch, so it won't risk overriding your own changes.
